hyperapp.d.ts

```diff
@@ -0,0 +1,45 @@
+export as namespace hyperapp
+
+// VNode part
+
+export interface VNode<Data> {
```

Comment:
Yes using : 
```tsx
const link = h('a', {})
```

Will do not raise anything. Thats why we are using `VNode<{}>` as default. Instead of `any`.